src/components/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -206,12 +220,16 @@ export default function ConfigPanel({
                 </div>
               ) : (
                 <div className="config-panel__no-description u-vertically-center">
-                  <div className="config-panel__no-description__message">
-                    <img src={bulbImage} alt="lightbulb" />
-                    <p>
+                  <div className="config-panel__message">
+                    <img
+                      src={bulbImage}
+                      alt="lightbulb"
```

Comment:
Given this image is purely decorative and adds no semantic value, this alt should be blank.

Reply:
I thought so as well but the linter won't let it go without defining one.